libraries/plugins/xfpga/reconf.c

```diff
@@ -159,20 +158,15 @@ STATIC fpga_result open_accel(fpga_handle handle, fpga_handle *accel)
 	if (matches == 0) {
 		OPAE_ERR("No accelerator found to reconfigure");
 		result = FPGA_BUSY;
-		goto destroy_token;
+		goto free_props;
 	}
 
-	result = xfpga_fpgaOpen(token, accel, 0);
+	result = xfpga_fpgaOpen(*token, accel, 0);
 	if (result != FPGA_OK) {
 		OPAE_ERR("Could not open accelerator for given slot");
-		goto destroy_token;
+		goto free_props;
```

Comment:
Open fails , destroy token   xfpga_fpgaDestroyToken(&token)

Reply:
result = xfpga_fpgaOpen(token, accel, 0); Line 165 fails, accel is not valid 
line 405: if (accel && xfpga_fpgaClose(accel) != FPGA_OK) {
fpga_handle accel is not valid

Reply:
It's initialized to NULL on 266.